### ISSUE TITLE
Network trained with 4 GPUs and 54 epochs has worse performance

### ISSUE THREAD

**Issue by ZuoJiaxing** [zju3dv/NeuralRecon#53]
I trained Neucon on 4 RTX5000 (16G) for 54 epochs. However, the evaluations on the Scannetdataset test split are worse than your released model trained for only 47 epochs. 

Here are the results from my trained model (54 epochs) :
    AbsRel 0.072
   AbsDiff 0.117
     SqRel 0.037
      RMSE 0.207
   LogRMSE 0.120
        r1 0.927
        r2 0.960
        r3 0.973
  complete 0.861
     dist1 0.066
     dist2 0.179
      prec 0.568
     recal 0.349
    fscore 0.429

Here are the results from your released model (47 epochs):
    AbsRel 0.065
   AbsDiff 0.098
     SqRel 0.038
      RMSE 0.197
   LogRMSE 0.113
        r1 0.933
        r2 0.961
        r3 0.974
  complete 0.892
     dist1 0.053
     dist2 0.133
      prec 0.687
     recal 0.473
    fscore 0.558

Do you have any clues for this? Thanks a lot!

**Comment by TheRealPoseidon** [comment]
May I ask if you train the neucon on the whole ScanNet data set (with over 1Tb in size) or the preprocessed version (about 5.6Gb)?

**Comment by JiamingSuen** [comment]
> May I ask if you train the neucon on the whole ScanNet data set (with over 1Tb in size) or the preprocessed version (about 5.6Gb)?

Please specify which "preprocessed version" you are referring to.

**Comment by TheRealPoseidon** [comment]
> > May I ask if you train the neucon on the whole ScanNet data set (with over 1Tb in size) or the preprocessed version (about 5.6Gb)?
> 
> Please specify which "preprocessed version" you are referring to.

Hi Jiaming Suen,
Thanks for reply and your impressive work on 3D reconstruction. May I ask, what's the size of scannet you used for training NeuralRecon. The entire scannet_v2 (1.2TB) or the 5.6GB "scannet_frames_25k.zip"? 
p.s.  In the python script provided by Prof. Dr. Angela Dai, "PREPROCESSED_FRAMES_FILE" refers to the 5,6GB training set. Sorry,I didn't make it clear.
Best regards,
Po

**Comment by JiamingSuen** [comment]
It should be scannet_v2 (1.2TB).

**Comment by weihaosky** [comment]
I have the similar problem. I trained the network on 8 GTX 2080 Ti with batch size of 1 on each GPU. Then I tested the results of models trained after 25, 30, 35, 40, 45, 50, 60, 70 epochs. The best model only gets a performance of abe_rel=0.068 and fscore=0.482, which is far from the results in your paper.

Also, the results from your released model (47 epochs) is:
    AbsRel 0.065
   AbsDiff 0.099
     SqRel 0.038
      RMSE 0.197
   LogRMSE 0.113
        r1 0.932
        r2 0.961
        r3 0.975
  complete 0.892
     dist1 0.053
     dist2 0.135
      prec 0.687
     recal 0.471
    fscore 0.557
which is the same as that reported by @ZuoJiaxing while different from that in the paper.

May I ask the training setup of yours? e.g. the batch size, the numbers of GPUs, the learning rate
Many thanks!
